### PR TITLE
tidb-lightning-faq: Add item for copying a schema

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Add item about copying a schema (both definition and data).

This is sometimes requested for testing/acceptance/etc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-faq.md [hunk 1]
```diff
@@ -205,3 +205,49 @@ The steps are as follows:
...
+Then we create a `/tmp/tidb-lightning.toml` file with the following content:
+
+```
+[tidb]
+host = "127.0.0.1"
+port = 4000
+user = "root"
+
+[tikv-importer]
+backend = "local"
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-faq.md:228]: maybe for simplicity (for example lightning's node can't access TiKV) we should recomment "tidb" backend

**Review by lance6716 — APPROVED** [review]
rest lgtm

File: tidb-lightning/tidb-lightning-faq.md [hunk 2]
```diff
@@ -205,3 +205,48 @@ The steps are as follows:
...
+type = 'table-schema'
+```
+
+Here we set `schema = 'test2'` as we want to use a different schema name than the one that we had in the original dump. The name of the table is taken from the filename.
+
+Then we use this config file to run the import.
+
+```
+tiup tidb-lightning -config /tmp/tidb-lightning.toml
+```
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-faq.md:252]:
```suggestion
3. Use this configuration file to run the import.

    ```
    tiup tidb-lightning -config /tmp/tidb-lightning.toml
    ```
```

**Review by ran-huang — COMMENTED** [review]
Minor style and format issues. Rest LGTM.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.6 label because the v6.6 docs have been archived at https://docs-archive.pingcap.com/tidb/v6.6 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-7.0 label because the v7.0 docs have been archived at https://docs-archive.pingcap.com/tidb/v7.0 and will no longer receive new updates.

File: tidb-lightning/tidb-lightning-faq.md [hunk 3]
```diff
@@ -209,3 +209,48 @@ The steps are as follows:
 2. Configure the required labels for TiKV and PD.
 3. Create the placement rule policy and apply the created policy to the target table.
 4. Use TiDB Lightning to import data into the target table.
+
+## How can I use TiDB Lightning and Dumpling to copy a schema
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-faq.md:213]:
```suggestion
## How can I use TiDB Lightning and Dumpling to copy a schema?
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**